CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ project(MAYO VERSION 1.0 LANGUAGES C CXX ASM)
 set(MAYO_SO_VERSION "0")
 set(CMAKE_C_STANDARD 99)
 
+set_property(GLOBAL PROPERTY CTEST_TARGETS_ADDED 1)
```

Comment:
What is the purpose of this property?

Reply:
I was not using CTest so wanted to hide targets when came with it. Should be removed.